.gitignore

```diff
@@ -16,6 +16,7 @@
 # Ignore cache
 /.sass-cache
 /.cache
+/source/github_api_output.json
```

Comment:
The file is generated on root level, so this needs to be `github_api_output.json` (without the `source/` dir)